backend/src/packages/meditation/meditation.service.ts

```diff
@@ -43,10 +43,11 @@ class MeditationService implements Service {
     name: string;
     file: FileUploadRequestDto;
   }): Promise<MeditationEntryCreateResponseDto> {
-    const { url, contentType } = await this.fileService.create(file);
+    const { id, url, contentType } = await this.fileService.create(file);
     const item = await this.meditationRepository.create(
       MeditationEntity.initializeNew({
-        name: name,
+        id,
```

Comment:
Why are you using `id` of the file to create a meditation?

Reply:
Because I thought they should be added to the meditation entries

Reply:
Farid is right, id's are created automatically, you task is to return it in response not to set it

Reply:
Yes, sorry, you are correct, I got it misunderstood